static/js/classes/doc.js

```diff
@@ -7,7 +7,27 @@ class Doc {
     this.annotations = [];
   }
 
-  getAnnotationRanges() {
-    return this.annotations.map(function (annotation) { return annotation.getRangeAsArray(); });
+  // getAnnotationRanges() {
+  //   return this.annotations.map(function (annotation) { return annotation.getRangeAsArray(); });
+  // }
```

Comment:
Is this commented out because its going to be used later but interferes with current functionality or should it be removed?

Reply:
it should be removed, we do that with something else now